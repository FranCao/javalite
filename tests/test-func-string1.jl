int main()
{
    string s = "Hello World";
    string r = reverse(s);
    print(s);
    print(r);
    r = reverse(r);
    print(r);

    s = "Foo Foo Doo Bar";
    string u = upper(s);
    print(s);
    print(u);
}