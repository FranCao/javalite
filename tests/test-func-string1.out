Hello World
dlroW olleH
Hello World
Foo Foo Doo Bar
FOO FOO DOO BAR